lina-acosta/tarea-4/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Calculator with JavaScript</title>
+    <link type="text/css" href="styles.css" rel="stylesheet">
+    <link rel="stylesheet" href="flex.css" />
+  </head>
+  <body onload="init();">
```

Comment:
@LinaAcosta , this is a security issue, please use an external js file to call the init method, you can add an event listener to the body